tools/clang/tools/dxcvalidator/dxcvalidator.cpp

```diff
@@ -155,24 +222,16 @@ runRootSignatureValidation(IDxcBlob *Shader,
 uint32_t hlsl::validate(
     IDxcBlob *Shader,            // Shader to validate.
     uint32_t Flags,              // Validation flags.
+    bool IsInternalValidator,    // Run internal validator.
```

Comment:
I don't think we should need this flag, ultimately.  But that will require a cleanup of how we do internal validation.

Reply:
In fact, it appears that the only way this flag is used is to ignore `DxcValidatorFlags_ModuleOnly` if it's not set.  I don't think this is right.  See my comment on the use of this flag in `hlsl::validateWithDebug`.